raiden_contracts/data/gas.json

```diff
@@ -1,17 +1,18 @@
 {
-    "MonitoringService.claimReward": 41927,
-    "MonitoringService.monitor": 207024,
-    "OneToN.claim": 92786,
-    "SecretRegistry.registerSecret": 46828,
+    "CustomToken.mint": 49423,
```

Comment:
This is the new line I wanted.

The other changes are not intended.  The gas values are somehow flaky.